guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+
+#### Timelocks
+
+Timelocks use the passing of time to define additional rules for how funds can be spent from a wallet.
+
+- Unlock additional spending conditions after a certain amount of time has passed.
+- Restrict individual transactions to not be processed immediately but only at a certain point of time in the future (pre-signed transactions).
+
+#### Veto keys
+
+In a multi-key setup it is possible to define a key hierarchy. This means that you can define keys that are required in every transaction. This technique is sometimes also called a “sovereign veto”, as it grants the holder of a certain key the power to veto any transaction, even if the required amount of signatures would have been otherwise reached.
+
+#### Powered by Miniscript
+
+Wallets that support these kinds of configurations typically rely on [Miniscript](/guide/glossary/#miniscript) to implement them. Miniscript is a simplified version of Bitcoin Script which helps to reduce the complexity of Bitcoin Script, the native programming language for Bitcoin. Miniscript is easier to read by developers, and also allows for various build-tools to help ensure that scripts are safe, valid, and efficient.
+
+## Practical example
+
+Imagine a multi-key wallet with a 2-of-3 signing threshold that should turn into a 1-of-3 after six months of inactivity. In order to have an additional safety layer, we also want the wallet to have a fourth key which should activate after one year.
+
+This wallet has three layers of spending conditions and uses relative timelocks to unlock layers two and three: 
+
+- **Condition 1 (primary):** The primary spending condition defines that two out of three keys are required to sign a transaction.
+- **Condition 2 (recovery):** After 6 months of wallet inactivity, unlock a second spending condition which only requires one of the three keys to sign transactions.
+- **Condition 3 (emergency):** After 12 months of wallet inactivity, unlock a third spending condition which activates a fourth key that is not part of the first two conditions.
+
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule.png"
+   retina = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule@2x.png"
+   mobile = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule-mobile@2x.png"
+   alt-text = "An illustration showing the unlocking scheme of the spending conditions."
+   width = 1600
+   height = 800
+%}
+
+</div>
+
+
+One important thing to note is that the conditions are additive. This means that, once unlocked, any given condition remains active even if additional one gets unlocked. In the above scenario, you can sign a transaction with two keys (condition 1) but also with any one key after six months (condition 2).
```

Comment:
```suggestion
One important thing to note is that previous conditions remain active and additional conditions are added to previous ones. This means that, once unlocked, any given condition remains active even if additional one gets unlocked. In the above scenario, you can sign a transaction with two keys (condition 1) but also with any one key after six months (condition 2).
```

Reply:
How about this? Would be shorter and more concise, I think. 

```suggestion
One important thing to note is that, once unlocked, spending conditions always remain active even if additional ones get unlocked. In the above scenario, you can sign a transaction with two keys (condition 1) but also with any one key after six months (condition 2).
```

Reply:
Also reads much better.